### PR TITLE
add docker-compose.yml for wiring up zipkin containers

### PR DESCRIPTION
**Pull request by zsol**

### CODE REVIEW

**Comment by kristofa** [comment]
@zsol I tried your branch locally and `docker-compose up` worked for me (Mac osx using boot2docker).
I sent some traces to `zipkin-collector` and could see them in `zipkin-web`.

Once the ROOTURL environment variable remark is fixed this is good to merge I would think :+1: 

Wdyt @rtyler @eirslett ?

File: deploy/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  links:
+    - cassandra:db
+web:
+  image: itszero/zipkin-web
+  links:
+    - query
+  ports:
+    - 8080:8080
+  environment: {
+    ROOTURL: "http://172.17.42.1:8080"
```
Comment (kristofa) [inline deploy/docker-compose.yml]: @zsol I don't know why the `ROOTURL` environment variable is needed for `zipkin-web`. If I remove it everything still works. Also it is a hard coded private ip address...  I guess we should remove it.

Can you also try if everything still works in your environment when you remove it?

Comment (zsol) [inline deploy/docker-compose.yml]: Yep, it's fine if I remove it. I never looked into what ROOTURL is used for, I just assumed [this](https://github.com/openzipkin/docker-zipkin/blob/master/deploy/deploy.sh#L5) is there for a good reason :)

**Comment by eirslett** [comment]
+1

**Pull request merged**